builtin/plugins/dposv3/dpos.go

```diff
@@ -1450,45 +1484,34 @@ func getDowntimeRecord(ctx contract.StaticContext, statistic *ValidatorStatistic
 }
 
 // only called for validators, never delegators
-func SlashInactivity(ctx contract.Context, validatorAddr []byte) error {
+func SlashInactivity(ctx contract.Context, statistic *ValidatorStatistic) (*ValidatorStatistic, error) {
```

Comment:
Seeing as the input `statistic` is modified directly there's no need to return it.

Reply:
good point, made change.